tailwind.config.ts

```diff
@@ -8,11 +15,14 @@ const config: Config = {
     './src/stories/**/*.{js,ts,jsx,tsx,mdx}',
   ],
   theme: {
+    fontWeight: { ...FONT_WEIGHT },
     extend: {
-      backgroundImage: {
-        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-        'gradient-conic': 'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
-      },
+      backgroundImage: { ...BACKGROUND_IMAGE },
+      backgroundColor: { ...COLORS },
+      colors: { ...COLORS },
+      fontSize: { ...FONT_SIZE },
+      lineHeight: { ...LINE_HEIGHT },
+      boxShadow: { ...BOX_SHADOW },
```

Comment:
해당 요소들을 tailwindcss Configuration 과 동일한 네이밍컨벤션(카멜케이스)를 사용해서  아래와 같이 작성하는것도 좋을 것 같습니다!

```suggestion
      extend: {
        fontWeight,
        backgroundImage,
        backgroundColor
        colors,
        fontSize,
        lineHeight,
        boxShadow
    }
```

Reply:
네. 해당 사항 반영해서 다시 올렸습니다.